guide/C/Makefile.am

```diff
@@ -25,10 +25,6 @@ entities = \
 	ch_currency.xml \
 	ch_dep.xml \
 	ch_bus_intro.xml \
```

Comment:
If we decide to keep the files merged (I personally don't have a strong opinion on this matter), I would like you to rename the remaining file as well. It now covers more than the introduction to the business features.